src/main/java/net/snowflake/ingest/streaming/internal/BlobMetadata.java

```diff
@@ -14,23 +14,23 @@
 class BlobMetadata {
   private final String path;
   private final String md5;
-  private final Constants.BdecVerion bdecVersion;
+  private final Constants.BdecVersion bdecVersion;
```

Comment:
Nit, for some of these changes it might make sense to pull them out into their own PR that we can quickly merge/review, it'll make reviewing this a bit easier.